src/main/java/com/palantir/gradle/revapi/RevapiPlugin.java

```diff
@@ -79,10 +79,12 @@ public void apply(Project project) {
 
                     task.getAcceptedBreaks().set(acceptedBreaks(project, configManager, extension.oldGroupAndName()));
 
-                    FileCollection thisJarFile = project.getTasks()
-                            .withType(Jar.class)
-                            .getByName(JavaPlugin.JAR_TASK_NAME)
-                            .getOutputs()
+                    // we don't want to just grab the output of the 'jar' task, because peiple using
+                    // 'com.palantir.shadow-jar' actually publish the output of a different task: 'shadowJar'
+                    FileCollection thisJarFile = project.getConfigurations()
+                            .getByName(JavaPlugin.RUNTIME_ELEMENTS_CONFIGURATION_NAME)
+                            .getOutgoing()
+                            .getArtifacts()
                             .getFiles();
```

Comment:
here's where com.palantir.shadow-jar sets this stuff up:

https://github.com/palantir/gradle-shadow-jar/blob/4cfdc85663f7262b688d80a329fc21686e2b72fd/src/main/groovy/com/palantir/gradle/shadowjar/ShadowJarPlugin.java#L246-L255